app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    name_f = command.split()[1]
+    new_f = command.split()[2]
```

Comment:
Consider using more descriptive variable names instead of 'name_f' and 'new_f'. According to the checklist, variable names should be descriptive and correct.